lib/plugins/openaps.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+var _ = require('lodash');
+var moment = require('moment');
+
+var timeago = require('./timeago')();
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['timeago' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1432442962/issues/source?bid=2452379&fileBranchId=2989096#l6)